host/src/FunctionsCustomHost/FunctionsCustomHost.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
I think it is safe to use net9.0 since it is already RC.

Reply:
I receive this error when trying to do so:
![image](https://github.com/user-attachments/assets/ba7543e7-48c0-4701-a2a5-decb75db89b2)

Reply:
nvm fixed it